CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## 0.5.6dev
 
+* [Feature] Shows missing driver package suggestion message ([#124](https://github.com/ploomber/jupysql/issues/124))
```

Comment:
FYI: there's no need to add the link manually. our deployment link takes care of it. just do (#NUMBER)